sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -42,6 +42,8 @@ export interface EducationProgramDto {
   hasIntlExchange?: string;
   intlExchangeProgramEligibility?: string;
   programDeclaration: boolean;
+  hasOfferings?: boolean;
```

Comment:
Is there a reason for this being optional?

Reply:
EducationProgramDto is used elsewhere and hasOfferings attribute is not populated. 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/72251620/168879758-a5a3e18f-1028-47ae-878e-23693f2671b4.png">

Reply:
I would recommend to create a new DTO(using a copy of existing) for getProgram because the other place where EducationProgramDto is used is createProgram and updateProgram where hasOfferings which totally not relevant. 
Also during re-organizing API these properties may get copied to APIInDTOs.